ci/test_wheel_dask_cudf.sh

```diff
@@ -41,6 +41,7 @@ pushd python/dask_cudf/dask_cudf
 DASK_DATAFRAME__QUERY_PLANNING=True python -m pytest \
   --junitxml="${RAPIDS_TESTS_DIR}/junit-dask-cudf.xml" \
   --numprocesses=8 \
+  --dist=worksteal \
```

Comment:
Curious how much of a speedup you see with this change?

Reply:
When we switched cudf wide, we were able to cut down 25% of the execution time: https://github.com/rapidsai/cudf/pull/15207